packages/material-ui-lab/src/TabPanel/TabPanel.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = {
+  /* Styles applied to the root element. */
+  root: {},
+};
+
+const TabPanel = React.forwardRef(function TabPanel(props, ref) {
+  const { activeValue, children, className, classes, value, ...other } = props;
+
+  return (
+    <Box
```

Comment:
What do you think of replacing the Box with a div? We are not using the Box in any of the public components, yet.

Reply:
Why was it used in the demos?

Reply:
It's used in the demo to reduce the boilerplate, but it comes with the additional issues the Box has, on top of the current issues the styling solution has. Do you want me to link these issues for completeness?

Reply:
> Do you want me to link these issues for completeness?

Would be great for future readers :)

Reply:
Here is what come to mind:

- About x4 slower: https://github.com/mui-org/material-ui/blob/master/packages/material-ui-benchmark/README.md#material-uistyles (`Box x 3,850 ops/sec ±3.58% (178 runs sampled)` vs `WithStylesButton x 16,844 ops/sec ±0.34% (191 runs sampled)`) #16111.
- The runtime cost of the style scale linearly with the number of instances #16756.
- The injected style aren't editable in the dev tools.
- The injected styled comes from a nonstable class name.

Reply:
If it has problems then we shouldn't use it in the demos. It's troubling that we recommend components that we wouldn't event use internally.

Reply:
I think that the key element to take into account is that the usage of the Box component is optional for the developers, using it in the core component would **force** them to.

I don't think that it's ready for prime-time yet. However, it was interesting to release it as some developers don't bother and for us to collect more feedback.

Reply:
>I don't think that it's ready for a systematic usage yet.

So it shouldn't have been release in the core.

Reply:
Done